icons/Wttj/content.json

```diff
@@ -0,0 +1,6 @@
+{
+  "width": 15,
+  "height": 15,
+  "viewBox": "0 0 599 500",
+  "block": "<path d=\"M142.672326,484.011129 C140.714747,491.909083 140.158005,497.55612 143.171599,499.24341 C146.393523,501.04917 152.043563,500.029616 157.786992,491.682914 C208.551149,417.890895 262.095427,310.852079 311.020537,220.743605 C300.226912,251.990782 292.759377,275.077938 287.673263,311.516225 C276.944293,388.460246 301.642116,448.585216 321.96502,476.709112 C332.359945,491.090568 341.788651,501.767166 371.040989,497.408931 C520.237251,475.20132 564.38515,343.843987 582.754066,277.221155 C616.2592,155.768574 591.730195,87.2502334 564.945484,43.0145114 C537.420845,-2.45257337 457.447453,-22.3482373 399.240904,36.3586902 C395.993837,39.7584004 391.708714,50.11549 404.643104,64.0840985 C431.718757,92.9224003 571.38574,247.578698 515.722248,375.446572 C493.114903,427.386389 430.863888,413.008523 400.325654,361.617973 C367.678982,306.676933 352.959424,200.155075 373.505025,99.5207818 C374.000705,97.093956 374.988474,92.0320855 369.417456,89.9391277 C365.893813,88.6144254 362.054085,88.8908538 358.41191,94.7461097 C356.917685,97.1513956 305.661439,170.06746 246.140259,278.904855 C272.813621,151.424699 209.858596,16.689195 199.197872,10.8195792 C196.155542,9.14664897 195.652678,7.32652959 189.528509,7.32652959 L9.81915583,7.32652959 C-2.28909944,7.32652959 -1.22590052,17.6872092 2.59586857,21.4459172 C48.5720381,76.1177083 198.486678,258.998422 142.672326,484.011129\" fill=\"#000\" fill-rule=\"evenodd\"/>"
```

Comment:
Do we want fill to always be black or should it be `currentColor`?

Reply:
We need currentColor :)

Reply:
Should be currentColor 🤔 I think the script has failed to replace the fill attribute.

Reply:
Yes, because we check another color, maybe we need to replace any color on fill attribute

Reply:
I think it's because of this part of the script
```
svgContent = svgContent[1].replace(/fill="#134B45"/g, 'fill="currentColor"').trim()
```

Reply:
Yes, can you change this rule pls ? :)

Reply:
fixed !